test/Api/CheckoutApiTest.php

```diff
@@ -23,11 +23,22 @@
  */
 class CheckoutApiTest extends \PHPUnit_Framework_TestCase
 {
+    private static $api;
+    private static $test_accounts;
+    private static $location_id;
 
     /**
      * Setup before running each test case
      */
     public static function setUpBeforeClass() {
+        self::$api = new \SquareConnect\Api\CheckoutApi();
+        self::$test_accounts = new \SquareConnect\TestAccounts();
+        // Configure OAuth2 access token for authorization: oauth2
+        $account = self::$test_accounts->{'US-Prod-Sandbox'};
+        $access_token = $account->{'access_token'};
+        Configuration::getDefaultConfiguration()->setAccessToken($access_token);
+
+        self::$location_id = $account->{'location_id'};
```

Comment:
```php
self::$location_id = $account->location_id;
```

PHP magic getter works with this syntax. You should avoid using the `->{}` syntax where possible